### DISCUSSION TITLE
To postpone multi camera attachment

### DISCUSSION THREAD

**Discussion by leo150** [HaishinKit/HaishinKit.swift#1327] [Q&A]
When the multi camera is attached from the app start, it works fine. However, attaching it later produces no image from the second camera. My goal is to be able to attach the multi camera when the user presses a button. 

I saw a comment a comment for `attachMultiCamera` method that should be related:
> - Warning: This method can't use appendSampleBuffer at the same time.

Is it achievable to postpone the multi camera attachment? Thanks.

**Comment by shogo4405** [comment]
I think it should work with the current version, which is in the 16.x series. Doesn't it work well? I noticed that it doesn't work properly with the 15.x series, so I'm thinking about making some fixes.

**Comment by shogo4405** [comment]
Starting from version 1.7.2, it is now possible to set it later. If there is a possibility of using multiple cameras, you need to initialize it as follows:

```
// If you're using multi-camera functionality, please make sure isMultiCamSessionEnabled = true. Initialization point.
stream.isMultiCamSessionEnabled = true
```
